msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md

```diff
@@ -248,19 +251,19 @@ Apps shared to the meeting stage vary in size based on the state of the meeting
 
 #### Meeting stage (without side panel)
 
-When the side panel isn't open, the meeting stage is 994x678 pixels by default and can be a minimum 792x382 pixels.
+When the side panel isn't open, the meeting stage is 994x678 pixels by default and can be a minimum 792x382 pixel.
 
 :::image type="content" source="~/assets/images/apps-in-meetings/meeting-stage-no-side-panel.png" alt-text="Image showing shared meeting stage responsiveness with the side panel closed." border="false":::
 
 #### Meeting stage (with side panel)
 
-When the side panel is open, the meeting stage is 918x540 pixels by default and can be a minimum 472x382 pixels.
+When the side panel is open, the meeting stage is 918x540 pixels by default and can be a minimum 472x382 pixel.
```

Comment:
Please check for singular/ plural, the word pixel is used in plural, please make it plurals.